index.html

```diff
@@ -1,5 +1,3 @@
----
```

Comment:
These are required for jekyll.

Reply:
@abishekvashok Oh okay. And as for the icons not rendering, please let it load completely.

Reply:
@abishekvashok Fixed.